config/crossSections13p6TeV.yaml

```diff
@@ -0,0 +1,718 @@
+  unc: 11.46
+QCD_PT-470to600_MuEnrichedPt5:
+  crossSec: 68.24
+  reference: XSDB QCD_PT-470to600_MuEnrichedPt5_TuneCP5_13p6TeV_pythia8 modifiedOn 2024-04-03 19:47:45
+  unc: 0.2049
+QCD_PT-600to800_MuEnrichedPt5:
+  crossSec: 21.37
+  reference: XSDB QCD_PT-600to800_MuEnrichedPt5_TuneCP5_13p6TeV_pythia8 modifiedOn 2023-05-26 21:19:13
+  unc: 0.199
+QCD_PT-800to1000_MuEnrichedPt5:
+  crossSec: 3.913
+  reference: XSDB QCD_PT-800to1000_MuEnrichedPt5_TuneCP5_13p6TeV_pythia8 modifiedOn 2023-03-26 13:58:47
+  unc: 0.03526
+QCD_PT-1000_MuEnrichedPt5:
+  crossSec: 1.323
+  reference: XSDB QCD_PT-1000_MuEnrichedPt5_TuneCP5_13p6TeV_pythia8 modifiedOn 2024-04-03 19:47:38
+  unc: 0.003921
+
+QCD_PT_15to30:
+  crossSec: 1301000000.0
+  reference: XSDB QCD_PT-15to30_TuneCP5_13p6TeV_pythia8 modifiedOn 2024-04-03 20:23:32
+  unc: 2135000.0
+QCD_PT_30to50:
+  crossSec: 113300000.0
+  reference: XSDB QCD_PT-30to50_TuneCP5_13p6TeV_pythia8 modifiedOn 2023-03-26 13:58:27
+  unc: 589300.0
+QCD_PT_50to80:
+  crossSec: 16760000.0
+  reference: XSDB QCD_PT-50to80_TuneCP5_13p6TeV_pythia8 modifiedOn 2024-04-03 19:47:46
+  unc: 27450.0
+QCD_PT_80to120:
+  crossSec: 2534000.0
+  reference: XSDB QCD_PT-80to120_TuneCP5_13p6TeV_pythia8 modifiedOn 2023-03-26 13:58:49
+  unc: 13280.0
+QCD_PT_120to170:
+  crossSec: 445800.0
+  reference: XSDB QCD_PT-120to170_TuneCP5_13p6TeV_pythia8 modifiedOn 2023-03-26 13:58:29
+  unc: 2174.0
+QCD_PT_170to300:
+  crossSec: 113700.0
+  reference: XSDB QCD_PT-170to300_TuneCP5_13p6TeV_pythia8 modifiedOn 2023-03-26 13:58:34
+  unc: 547.6
+QCD_PT_300to470:
+  crossSec: 7559.0
+  reference: XSDB QCD_PT-300to470_TuneCP5_13p6TeV_pythia8 modifiedOn 2023-05-26 21:19:12
+  unc: 36.67
+QCD_PT_470to600:
+  crossSec: 626.4
+  reference: XSDB QCD_PT-470to600_TuneCP5_13p6TeV_pythia8 modifiedOn 2023-03-26 13:58:44
+  unc: 2.863
+QCD_PT_600to800:
+  crossSec: 178.6
+  reference: XSDB QCD_PT-600to800_TuneCP5_13p6TeV_pythia8 modifiedOn 2023-03-26 13:58:45
+  unc: 0.8268
+QCD_PT_800to1000:
+  crossSec: 30.57
+  reference: XSDB QCD_PT-800to1000_TuneCP5_13p6TeV_pythia8 modifiedOn 2023-03-26 13:58:47
+  unc: 0.1405
+QCD_PT_1000to1400:
+  crossSec: 8.92
+  reference: XSDB QCD_PT-1000to1400_TuneCP5_13p6TeV_pythia8 modifiedOn 2023-03-26 13:58:26
+  unc: 0.04204
+QCD_PT_1400to1800:
+  crossSec: 0.8103
+  reference: XSDB QCD_PT-1400to1800_TuneCP5_13p6TeV_pythia8 modifiedOn 2023-03-26 13:58:30
+  unc: 0.003798
+QCD_PT_1800to2400:
+  crossSec: 0.1148
+  reference: XSDB QCD_PT-1800to2400_TuneCP5_13p6TeV_pythia8 modifiedOn 2023-03-26 13:58:35
+  unc: 0.0005556
+QCD_PT_2400to3200:
+  crossSec: 0.007542
+  reference: XSDB QCD_PT-2400to3200_TuneCP5_13p6TeV_pythia8 modifiedOn 2023-03-26 13:58:36
+  unc: 3.779e-05
+QCD_PT_3200:
+  crossSec: 0.0002331
+  reference: XSDB QCD_PT-3200_TuneCP5_13p6TeV_pythia8 modifiedOn 2023-03-26 13:58:43
+  unc: 1.208e-06
+
+ST_s-channel_antitop:
+  crossSec: '0.0'
```

Comment:
take from XSDB

Reply:
Fixed in later commit (I don't know how I forgot this one too :sad:)